config/kustomize/patches/deployment-resources.yaml

```diff
@@ -7,6 +7,12 @@ metadata:
 spec:
   template:
     spec:
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      tolerations:
+        - key: "node-role.kubernetes.io/master"
```

Comment:
`master` is the old name. the new one should be something like `control-plane`

Reply:
`node-role.kubernetes.io/control-plane`

Reply:
Changed in https://github.com/giantswarm/cluster-api-provider-cloud-director-app/pull/30/commits/1dd91d022c77e55268a1cf6109826fad43fcd137